src/devices/Mbi5027/Mbi5027PinMapping.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device.Multiplexing
+{
+    /// <summary>
+    /// Represents pin bindings for the MBI5027.
+    /// </summary>
+    public struct Mbi5027PinMapping
+    {
+        /// <param name="sdi">Serial data in pin</param>
+        /// <param name="clk">Shift register pin (shift to data register)</param>
+        /// <param name="le">Register clock pin (latch)</param>
+        /// <param name="oe">Output enable pin</param>
+        /// <param name="sdo">Serial data out pin -- required for open circuit detection</param>
+        public Mbi5027PinMapping(int sdi, int clk, int le, int oe = -1, int sdo = -1)
+        {
+            Sdi = sdi;
+            Clk = clk;
+            LE = le;
+            OE = oe;
+            Sdo = sdo;
+        }
+
+        /// <summary>
+        /// Minimal pin bindings for the MBI5027.
+        /// </summary>
+        public static Mbi5027PinMapping Minimal => new Mbi5027PinMapping(16, 20, 21);
```

Comment:
this should be a method or return a cached instance (same below)

Reply:
actually if this is a struct then I'm ok with this

Reply:
Ya. I don't think there is much value of returning a cached instance (even if it was a class). Isn't there a minimum number of uses to expect to merit creating a cached instance? I suspect this property would get accessed <=1 times per app lifetime.